src/Formats/MySQLBlockInputStream.cpp

```diff
@@ -30,6 +30,29 @@ namespace ErrorCodes
     extern const int NOT_IMPLEMENTED;
 }
 
+StreamSettings::StreamSettings(const Settings & settings)
+{
+    max_block_size = settings.max_read_mysql_rows;
```

Comment:
There is already a setting `max_block_size`, which is used for this purpose and for all previously made tables. It is not possible to just make it a different setting.

Reply:
emm. I will modify `max_block_size` to `max_read_mysql_rows`

Reply:
Yes, you override it, and it means that setting `max_block_size` is not used (you set default to `DEFAULT_BLOCK_SIZE`), but it might be already used for `MySQL` tables. It is better to make default as `max_block_size` setting instead, i.e. at least check whether you setting is changed and if not - use `max_block_size`.